File: lib/AbstractREST.php
<?php

class AbstractREST
{
   	function __construct()
	{			
		global $app;
	
	}

	function run(){
    }

    static function show_non_auth_error(){
    	echo '401';

    	return false;
    }

	static function ShowUknownCmd()
	{
		global $app;		
		http_response_code(404);
		
		return false;
	}

	static function ShowNoData()
	{
		global $app;		
		http_response_code(404);
		return false;
	}

	static function ShowError()
	{
		global $app;		
		echo '403';
		return false;
	}
}
?>

File: web/code/Teplo500Web/rest/abstract_rest.py

from teplo500web import web_app

class AbstractREST:

	def __init__(self):
		web_app.CreateAndInit()

	def run(self):
		return True

	def show_non_auth_error(self):
		return '401'

	def ShowUknownCmd(self):
		return '403'

	def ShowNoData(self):
		return '404'

	def ShowError(self):		
		return '500'
